src/utils/bastion.yaml

```diff
@@ -9,6 +9,9 @@ Parameters:
   AdminAppImage:
     Type: String
     Default: 'public.ecr.aws/y7d9d7k6/public_dummy:0.1.0'
+  CCFBucketName:
```

Comment:
One of the problems with this script. Bucket names have to be globally unique. We can add a prompt for this later.
For now I've just moved it up to params so that we can change it easily.